specs/darwin/wifi_networks.table

```diff
@@ -0,0 +1,17 @@
+table_name("wifi_networks")
+description("OS X known/remembered Wi-Fi networks list.")
+schema([
+    Column("ssid", TEXT, "SSID octets of the network"),
+    Column("ssid_string", TEXT, "SSID String of the network"),
```

Comment:
Yeah, I know and there isn't a character set associated with SSID octets. SSID == SSIDString in all the cases I have seen. I like network_name better, renaming.